components/templates/ListOfSuites.tsx

```diff
@@ -1,200 +1,235 @@
-    color: theme.palette.secondary.light,
-  },
-  nameOfTestOrSuiteLight: {
-    paddingLeft: theme.spacing(4),
-    fontSize: "0.875rem",
-    textAlign: "left",
-    fontFamily: "Roboto",
-    fontWeight: 400,
-    wordBreak: "break-all",
-    whiteSpace: "pre-wrap",
-  },
-  backgroundGrey: {
-    backgroundColor: "#d6d6d6",
-  },
-  backgroundDarkGrey: {
-    backgroundColor: "#474747",
-  },
-  backgroundWhite: {
-    backgroundColor: "white",
-  },
-  backgroundDark: {
-    backgroundColor: theme.palette.secondary.main,
-    border: "1px grey solid",
-  },
-  testBackgroundDark: {
-    backgroundColor: theme.palette.secondary.main,
-  },
-  greyTick: {
-    color: "#d4d3d3"
-  },
+    root: {
+        width: "100%",
+        maxWidth: 3500
+    },
+    title: {
+        fontSize: "2em"
+    },
+    container: {
+        paddingTop: theme.spacing(4),
+        maxWidth: 3400,
+        paddingBottom: theme.spacing(4)
+    },
+    paper: {
+        padding: theme.spacing(2),
+        display: "flex",
+        overflow: "auto",
+        flexDirection: "column"
+    },
+    padding: {
+        marginBottom: "5px",
+        marginLeft: "80%",
+        color: "#353690"
+    },
+    nameOfTestOrSuiteDark: {
+        paddingLeft: theme.spacing(4),
+        fontSize: "0.875rem",
+        textAlign: "left",
+        fontFamily: "Roboto",
+        fontWeight: 400,
+        wordBreak: "break-all",
+        whiteSpace: "pre-wrap",
+        color: theme.palette.secondary.light
+    },
+    nameOfTestOrSuiteLight: {
+        paddingLeft: theme.spacing(4),
+        fontSize: "0.875rem",
+        textAlign: "left",
+        fontFamily: "Roboto",
+        fontWeight: 400,
+        wordBreak: "break-all",
+        whiteSpace: "pre-wrap"
+    },
+    backgroundGrey: {
+        backgroundColor: "#d6d6d6"
+    },
+    backgroundDarkGrey: {
+        backgroundColor: "#474747"
+    },
+    backgroundWhite: {
+        backgroundColor: "white"
+    },
+    backgroundDark: {
+        backgroundColor: theme.palette.secondary.main,
+        border: "1px grey solid"
+    },
+    testBackgroundDark: {
+        backgroundColor: theme.palette.secondary.main
+    },
+    greyTick: {
+        color: "#d4d3d3"
+    }
 }))
 
 type Props = {
-  children: SuiteAndTest[]
-  stats
+  test_run_id: number,
+  stats,
   darkMode: boolean
 }
 
+export const ListOfSuites = function (props : Props) {
+const {test_run_id, stats, darkMode} = props
+const {loading, noData, testSuites} = getTestSuites(test_run_id, stats.toString());
 
-export const ListOfSuites = function(props: Props) {
-  const { children, stats, darkMode } = props
-  const classes = useStyles(props)
-  const [testInfoSection, setTestInfoSection] = useState(["No test selected"])
-  const [highlightedTest, setHighlightedTest] = useState(0)
+const classes = useStyles(props)
+const [testInfoSection, setTestInfoSection] = useState(["No test selected"])
+const [highlightedTest, setHighlightedTest] = useState(0)
```

Comment:
What’s  up with the indentation? Let's run prettifier maybe?

Reply:
@opishcheiko I did a commit where all I did was to run the prettifier D: so the code should be formatted already

Reply:
doesn't look pretty 😢

Reply:
😿

Reply:
I just executed the default VS Code analyzer to format the code, there are some changes but not many, I guess its something we should configure and do at some point